go/vt/vtgate/vtgate.go

```diff
@@ -84,6 +85,11 @@ type RegisterVTGate func(vtgateservice.VTGateService)
 // RegisterVTGates stores register funcs for VTGate server.
 var RegisterVTGates []RegisterVTGate
 
+var (
+	// RPCErrorOnlyInReply informs vtgateservice(s) about how to return errors.
+	RPCErrorOnlyInReply = flag.Bool("rpc-error-only-in-reply", false, "if true, supported RPC calls from vtgateservice(s) will only return errors as part of the RPC server response")
```

Comment:
Please sync with Jeff, as he thinks we should pass the error back as return value as well.